🐛 Source Shopify: Fixed rate limits throttling (#5470)
airbytehq/airbyte: docs/integrations/sources/shopify.md

### Performance considerations

Shopify has some [rate limit restrictions](https://shopify.dev/concepts/about-apis/rate-limits).

> Typicaly, there shoulld not be issues with throttling or exceeding the rate limits, in some edge cases, user can receive the warning message as follows:

Reviewer: ```suggestion
Typically, there should not be issues with throttling or exceeding the rate limits but in some edge cases, user can receive the warning message as follows:
```

Author: Corrected.

```
"Caught retryable error '<some_error> or null' after <some_number> tries. Waiting <some_number> seconds then retrying..."
```

> It is normal when the connector hit the 429 - Rate Limit Exceeded HTTP Error.
> With given error message the sync operation is still goes on, but will require more time to finish.

Reviewer: ```suggestion
This is expected when the connector hits the 429 - Rate Limit Exceeded HTTP Error.
With given error message the sync operation still goes on, but will require more time to finish.
```

Author: Corrected.
